deployments/helm/hephaestus/templates/_helpers.tpl

```diff
@@ -8,21 +8,30 @@ Return the proper controller image name.
 {{- end }}
 
 {{/*
-Return the proper log processor image name.
+Return the controller standard labels.
 */}}
-{{- define "hephaestus.logprocessor.image" -}}
-{{- include "common.images.image" (dict "imageRoot" .Values.logProcessor.image "global" .Values.global) }}
+{{- define "hephaestus.controller.labels.standard" -}}
+{{- include "common.labels.standard" . }}
+app.kubernetes.io/component: controller
```

Comment:
we use `    {{- include "common.labels.component" "cainjector" | nindent 4 }}`

Reply:
I've been using the [bitnami common](https://github.com/bitnami/charts/tree/master/bitnami/common#labels) library chart for a while now and, unfortunately, they don't have that function available.

I _believe_ I wrote this function way back when and it does essentially the same thing.